src/test/java/org/spectrumauctions/sats/core/bidlang/xor/CatsXORTest.java

```diff
@@ -104,4 +107,82 @@ private void testNoCap(long seed) throws UnsupportedBiddingLanguageException {
+        double valueOfGoods = 0;
+
+        if (dir.isDirectory()) {
+            for (String fileName : dir.list()) {
+                try (BufferedReader br = new BufferedReader(new FileReader(path.toString() + "/" + fileName))) {
+                    String line;
+                    while ((line = br.readLine()) != null) {
+                        if (line.length() == 0) continue;
+                        if (line.substring(0, 4).equals("bids")) {
+                            numberOfBids += Integer.valueOf(line.substring(5));
+                        } else if (line.substring(0, 5).equals("dummy")) {
+                            numberOfBidders += Integer.valueOf(line.substring(6));
+                        } else {
+                            String[] fields = line.split("\t");
+                            try {
+                                int bidNr = Integer.valueOf(fields[0]); // To check that it can be read -> we're in a bid line
+                                valueOfGoods += Double.valueOf(fields[1]);
+                                numberOfGoods += (fields.length - 3);
+                            } catch (NumberFormatException e) {
+                                continue;
+                            }
+                        }
+                    }
+
+                }
+
+            }
+
+            double bidsPerBidderCats = (double) numberOfBids / numberOfBidders;
+            double avgBundleSize = (double) numberOfGoods / numberOfBids;
+            double valuePerGood = valueOfGoods / numberOfGoods;
+
+            int numberOfBidsSats = 0;
+            int numberOfBiddersSats = 0;
+            int numberOfGoodsSats = 0;
+            double valueOfGoodsSats = 0;
+
+            for (int i = 0; i < dir.list().length / 100; i++) {
+                CATSRegionModel model = new CATSRegionModel();
+                model.setNumberOfGoods(256);
+                model.setNumberOfBidders(25);
+                List<CATSBidder> bidders = model.createNewPopulation();
+                numberOfBiddersSats += bidders.size();
+
+                for (CATSBidder bidder : bidders) {
+                    CatsXOR valueFunction = bidder.getValueFunction(CatsXOR.class);
+                    Set<XORValue<CATSLicense>> bids = valueFunction.getCATSXORBids();
+                    numberOfBidsSats += bids.size();
+                    for (XORValue<CATSLicense> bid : bids) {
+                        numberOfGoodsSats += bid.getLicenses().size();
+                        valueOfGoodsSats += bid.value().doubleValue();
+                    }
+                }
+            }
+
+            double bidsPerBidderSats = (double) numberOfBidsSats / numberOfBiddersSats;
+            double avgBundleSizeSats = (double) numberOfGoodsSats / numberOfBidsSats;
+            double valuePerGoodSats = valueOfGoodsSats / numberOfGoodsSats;
+
+            // TODO: Find the reason why SATS generally has a bit lower prices, a bit less goods per bid,
```

Comment:
You do want to merge the TODO, don't you?

Reply:
No, that's alright :-) I just wanted to ensure you didn't forget something you wanted to merge with this PR